x-pack/solutions/search/plugins/enterprise_search/public/navigation_tree.ts

```diff
@@ -148,12 +148,6 @@ export const getNavigationTreeDefinition = ({
                       },
                       link: 'enterpriseSearchApplications:searchApplications',
                       renderAs: 'item',
-                      title: i18n.translate(
```

Comment:
From where nav gets translations now?

Reply:
It gets the translation from here: https://github.com/elastic/kibana/blob/ce5883c8837a10a2fce409227ca3cd1fa67415ea/x-pack/solutions/search/plugins/enterprise_search/public/plugin.ts#L123C5-L128C7

The I removed translation was overwriting the original.